nixos/modules/hardware/video/nvidia.nix

```diff
@@ -37,6 +37,22 @@ in
     ];
 
   options = {
+    hardware.nvidia.cuda.enable = lib.mkOption {
+      type = types.bool;
+      default = enabled;
+      defaultText = literalExpression ''elem "nvidia" config.services.xserver.videoDrivers'';
+      description = lib.mdDoc "Whether to enable cuda.";
```

Comment:
I'm not sure I, as a user, would understand this text. "Enable CUDA" in which sense? From the rest of the diff I see that this option adds some particular version of cuda shared libraries (in addition to `libcuda.so`, the userspace driver) into `/run/opengl-driver/lib` but then I'm not sure which use-case this is meant to address